apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/conf/Config.java

```diff
@@ -252,6 +252,25 @@ public static class Profile {
         public static int SNAPSHOT_TRANSPORT_BUFFER_SIZE = 500;
     }
 
+    public static class AsyncProfiler {
+        /**
+         * If true, skywalking agent will enable profile when user create a new async profiler task.
+         * Otherwise disable it.
+         */
+        public static boolean ACTIVE = true;
+
+        /**
+         * Max monitor time(second), if async profiler monitor time out of limit, then stop it.
+         * default 1h.
+         */
+        public static int MAX_DURATION = 600;
+
+        /**
+         * jfr directory generated by async profiler
+         */
+        public static String OUTPUT_PATH = "";
```

Comment:
It seems we don't have any default value for the output path

Reply:
This parameter is used when creating jfr files. If it is empty, Files.createTemp will be used. Otherwise, the file will be created in the specified directory.